source/CardGames.py

```diff
@@ -224,15 +207,31 @@ def start_game(self):
     dealer.dealCards(5, self.players)
     for player in self.players:
       player.showHand(player.hand)
-    
+
+  def check_if_empty(self):
+    if len(self.info_player_asked) == 0:
+          self.game_over()
+
+  def game_over(self):
+    j = 0
+    gm = ["G", "A", "M", "E", "O", "V", "E","R"]
+    for i in range(20):
+      print(" "*j, ".")
+      j += 3
+      if i == 10:
+        for i in range (len(gm)):
+          print(" "*j, gm[i])
+          j += 3
+
+    sys.exit()
```

Comment:
The pattern print is dope! I think it looks good and I look forward to playing it!

Reply:
Thanks! I'll try to improve it and maybe add a cool feature that displays the winner on the game over screen as well.